### ISSUE TITLE
PD-Panel

### ISSUE THREAD

**Issue by CuriaPLUS** [Lambda-IT/parlamentsdienste-components#70]
When using PD-Panel (see following example), the content is sometimes visible on load even though the collapsed prop is set to true

```
  <pd-panel collapsible :collapsed='true'>
    <pd-panel-header slot="header">{{ $t('person.group.detail.secretary.title') }}</pd-panel-header>
    <pd-panel-content>
      <cp-secretary />
    </pd-panel-content>
  </pd-panel>
```

or

```
  <pd-panel collapsible collapsed>
    <pd-panel-header slot="header">{{ $t('person.group.detail.secretary.title') }}</pd-panel-header>
    <pd-panel-content>
      <cp-secretary />
    </pd-panel-content>
  </pd-panel>
```

Result: 
![Screenshot 2021-08-02 at 16 44 31](https://user-images.githubusercontent.com/77280472/127879866-d35afeb5-e38c-43e6-8c9a-3f3cbe40057c.png)

It looks like the CSS Style is not always applied, sometimes it is: 
```
<div class="pd-panel-content-wrapper pd-panel-content-collapsed" aria-expanded="false" style="height: 0px; overflow: hidden;">
```

and sometimes the overflow: hidden is missing.

Furthermore the PD-Panel header cannot be clicked to toggle the collapse on Version 1.7.1

**Comment by matthiasbaldi** [comment]
Will be solved in August, Milestone 1.7.2

**Comment by matthiasbaldi** [comment]
@CuriaPLUS Can you please give me more information about how you feed this panel with data etc?
Because I was not able to reproduce the issue in our Stencil and Vue Demo app.
I think that it must have something to do with loading data, may there we have an edge case which is not covered yet.

**Comment by CuriaPLUS** [comment]
It does not happen every time, just sometimes when loading, the state of the dropdown is not set correctly.
We do not do anything special and the example above has no data loaded. It looks like sometimes the Styles are not correctly applied

![Screenshot 2021-08-09 at 09 34 43](https://user-images.githubusercontent.com/77280472/128672966-be2fcfb0-17e9-4921-834c-504c4745ef38.png)
![Screenshot 2021-08-09 at 09 34 37](https://user-images.githubusercontent.com/77280472/128672971-a664eb78-d0e4-4949-910e-c1c4625c1408.png)

**Comment by matthiasbaldi** [comment]
Hmmm thank you for your feedback. Am I right, that you experience the same behavior in the new alert with the collapse-feature?
Because the implementation is pretty the same.  Do you have the issue also there?

**Comment by CuriaPLUS** [comment]
We havent used that so far so cannot really say. Thanks for looking into it

**Comment by thomasschneiter** [comment]
Did you test this with v1.7.0? The 'hidden' style was missing and fixed together with the pd-alert feature in 1.7.0. Is it possible that this was the problem?

**Comment by thomasschneiter** [comment]
Since we are unable to reproduce this on our side, I've create some logs to debug the problem. Can you use [pre-release v1.7.2-1](https://github.com/Lambda-IT/parlamentsdienste-components/releases/tag/v1.7.2-1), check the console and report back what is logged when this bug happens?

**Comment by CuriaPLUS** [comment]
A possible solution might be to add some more styling to:
.pd-panel-content-collapsed { overflow: hidden } instead of applying the style to the element itself during initialisation

**Comment by thomasschneiter** [comment]
The `overflow: hidden` is applied to the element directly, because it needs to be applied during the expand animation and is only removed when it's fully expanded. The class `.pd-panel-content-collapsed` ignores the animation. It also depends if the panel starts expanded or collapsed that's why it's applied during initialization.

**Comment by CuriaPLUS** [comment]
Fixed with 1.7.2 (had some old version in Theme ❤️ )
